spec/factories/roles.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :role, class: Role do
```

Comment:
You don't need to specify the class here since it's the same as the factory name